### DISCUSSION TITLE
A real-time subtitle and translation program using Faster-whisper and translatepy for any language.

### DISCUSSION THREAD

**Discussion by mldljyh** [SYSTRAN/faster-whisper#275] [Show and tell]
[whisper_real_time_translation](https://github.com/mldljyh/whisper_real_time_translation)

I've created a real-time subtitle and translation program using faster-whisper, which is based on whisper, along with translatepy and tkinter.
There are still unresolved issues, but if you have any suggestions, please feel free to share them.

**Comment by guillaumekln** [comment]
Thanks for sharing.

Just a comment about your README page:

> Faster-Whisper also requires the command-line tool [ffmpeg](https://ffmpeg.org/) to be installed on your system, which is available from most package managers.

faster-whisper does not have this requirement (unlike the original Whisper implementation). It uses the Python module `av` which bundles the required ffmpeg libraries, so ffmpeg does not need to be installed on the system.

**Reply by mldljyh** [reply]
Thank you for providing the information. 

It appears that ffmpeg was required for the whisper_real_time, which serves as the foundation of this program, but it was not necessary for faster-whisper. I have immediately updated the README accordingly.
